eQuiz/modules/eQuiz.Web/Areas/Moderator/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,39 @@
+﻿
```

Comment:
We should not have a separate layout page in this area.
Use Layout.cshtml from the common area instead.